plugin/pkg/scheduler/scheduler.go

```diff
@@ -96,33 +96,34 @@ func (s *Scheduler) scheduleOne() {
 	}
 	metrics.SchedulingAlgorithmLatency.Observe(metrics.SinceInMicroseconds(start))
 
-	b := &api.Binding{
-		ObjectMeta: api.ObjectMeta{Namespace: pod.Namespace, Name: pod.Name},
-		Target: api.ObjectReference{
-			Kind: "Node",
-			Name: dest,
-		},
-	}
+	// Assume optimistically that the binding will succeed and send it
+	// in the background.
+	// The internal cache only assumes a pod anyway and waits for it to be
+	// read via watch, so that shouldn't cause problems.
+	assumed := *pod
+	assumed.Spec.NodeName = dest
+	s.config.SchedulerCache.AssumePodIfBindSucceed(&assumed, func() bool { return true })
+
+	go func() {
+		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
```

Comment:
It should - it's total e2e scheduling latency. If you want to just "scheduling algorithm latency" - you should use "SchedulingAlgorithmLatency" metric.